### PR TITLE
Update 11-tanweechong1973.md

### PR DESCRIPTION
**Pull request by tanweechong1973**

### CODE REVIEW

**Comment by tanweechong1973** [comment]
New pull Request 123 from weechong

**Comment by cody-carlson** [comment]
resolves #1 

- [ ] item 1
- [ ] item 2

File: _slides/11-tanweechong1973.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "123456wc push Welcome to our slide deck!"
```
Comment (cody-carlson-learning) [inline _slides/11-tanweechong1973.md:3]: This is looking like it could be ok!

Comment (cody-carlson-learning) [reply _slides/11-tanweechong1973.md:3]: ok we've talked and fixed this

**Review by cody-carlson-learning — APPROVED** [review]

File: _slides/11-tanweechong1973.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "123456wc push Welcome to our slide deck!!"
```
Comment (terkudok) [inline _slides/11-tanweechong1973.md:3]: This change is OK.

**Review by terkudok — APPROVED** [review]
Approved!

**Pull request merged**